ajax/libs/onecolor/3.0.1/one-color-all.map

```diff
@@ -0,0 +1,51 @@
+{
+  "version": 3,
+  "sources": [
+    "node_modules/browserify/node_modules/browser-pack/_prelude.js",
+    "/home/munter/git/one-color/index.js",
+    "/home/munter/git/one-color/lib/CMYK.js",
+    "/home/munter/git/one-color/lib/HSL.js",
+    "/home/munter/git/one-color/lib/HSV.js",
+    "/home/munter/git/one-color/lib/LAB.js",
+    "/home/munter/git/one-color/lib/XYZ.js",
+    "/home/munter/git/one-color/lib/color.js",
+    "/home/munter/git/one-color/lib/plugins/clearer.js",
+    "/home/munter/git/one-color/lib/plugins/darken.js",
+    "/home/munter/git/one-color/lib/plugins/desaturate.js",
+    "/home/munter/git/one-color/lib/plugins/greyscale.js",
+    "/home/munter/git/one-color/lib/plugins/lighten.js",
+    "/home/munter/git/one-color/lib/plugins/mix.js",
+    "/home/munter/git/one-color/lib/plugins/namedColors.js",
+    "/home/munter/git/one-color/lib/plugins/negate.js",
+    "/home/munter/git/one-color/lib/plugins/opaquer.js",
+    "/home/munter/git/one-color/lib/plugins/rotate.js",
+    "/home/munter/git/one-color/lib/plugins/saturate.js",
+    "/home/munter/git/one-color/lib/plugins/toAlpha.js"
```

Comment:
We noticed that one in the last production build. Caused by one of the browserify plugins that can't handle relative path source maps if I recall correctly. Ping @papandreou